rs/nervous_system/integration_tests/tests/sns_release_qualification.rs

```diff
@@ -267,4 +271,39 @@ fn test_sns_upgrade(sns_canisters_to_upgrade: Vec<SnsCanisterType>) {
             sns_governance_canister_id,
         );
     }
+
+    // Check the SNS canisters' wasm_memory_limits
+    // TODO(NNS1-3278): Move this check to before upgrading the canisters
```

Comment:
NCR. 

I'm not sure if it should be part of the release qualification test permanently. I think there are 2 things we want to test:

1. the one-off migration code works. For this purpose it seems OK to test in the release qualification test against mainnet WASMs.
2. all SNS initialization from now on creates framework canisters with correct wasm_memory_limit. IMHO this should be in a test case NOT part of the release qualification, since there is no need to test against mainnet WASMs. Are there already such tests?

Reply:
There are not already such tests I think, I don't know of how it would be tested before the changes in this MR. I edited the ticket and will edit the comment to say that this check should be moved there, for now I think it makes sense for it to live here as it will be useful for testing a future MR